index.d.ts

```diff
@@ -895,6 +896,34 @@ declare module "axoncore" {
         public delete(mID: string): Collection<Message>;
     }
 
+    interface CollectorOptions {
```

Comment:
I'm using this interface name already
```suggestion
    interface ReactionCollectorOptions {
```